build.gradle

```diff
@@ -42,31 +45,31 @@ repositories {
 }
 
 dependencies {
-    implementation('io.confluent:kafka-avro-serializer:6.2.2') {
+    implementation("io.confluent:kafka-avro-serializer:${confluentKafkaVersion}") {
         exclude group: 'org.apache.kafka', module:'kafka-clients'
     }
-    implementation('io.confluent:kafka-schema-serializer:6.2.2') {
+    implementation("io.confluent:kafka-schema-serializer:${confluentKafkaVersion}") {
         exclude group: 'org.apache.kafka', module:'kafka-clients'
     }
-    implementation 'io.confluent:common-config:6.2.2'
+    implementation "io.confluent:common-config:${confluentKafkaVersion}"
     implementation 'org.apache.avro:avro:1.11.0'
-    implementation('io.confluent:kafka-schema-registry-client:6.2.2') {
+    implementation("io.confluent:kafka-schema-registry-client:${confluentKafkaVersion}") {
         exclude group: 'org.apache.kafka', module:'kafka-clients'
     }
-    implementation('org.apache.kafka:kafka_2.12:2.8.1') {
+    implementation("org.apache.kafka:kafka_2.12:${apacheKafkaVersion}") {
         // contains kafka.utils.VerifiableProperties used by kafka-schema-registry-client, explicitly exclude
         // transitive dependencies
         transitive = false
     }
-    implementation 'io.confluent:common-utils:6.2.2'
+    implementation "io.confluent:common-utils:${confluentKafkaVersion}"
     implementation 'javax.ws.rs:javax.ws.rs-api:2.1.1'
     implementation 'org.glassfish.jersey.core:jersey-common:2.33'
     // given https://docs.confluent.io/current/installation/versions-interoperability.html matrix
-    // Confluent Platform 6.2.x is Apache Kafka 2.8.x
-    implementation 'org.apache.kafka:kafka-clients:2.8.1'
+    // Confluent Platform 7.3.x is Apache Kafka 3.3.x
```

Comment:
Nit: Move this comment to the definition of the version constants

Reply:
Fixed by 1ea9f318c00b9d23f77ddfee14bf87ee44764a38